### ISSUE TITLE
Create a single control that includes play/pause/step buttons

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/scenery-phet#561]
There are many places where we have Play/Pause and Step buttons together. And now we have a PDOM description that combines them like "Pause, play, or step forward in time while paused."

It seems like we should have a reusable type that combines these controls.

@samreid recommended that we look through sims to determine if there are any other patterns or controls that should be included in this group, and list out which sims could use this once it is made.

**Comment by jessegreenberg** [comment]
We are looking for controls that are arranged sort of like this: ![image](https://user-images.githubusercontent.com/6396244/70645977-fcd51200-1c13-11ea-938e-f5db711fe847.png)
 
I see them in
 - states-of-matter (SOMPlayPauseStepControl)

**Comment by jessegreenberg** [comment]
@pixelzoom pointed out on Slack that this already exists as TimeControlNode, which covers everything mentioned in this issue. We should now search for sims that should be using this component.

**Comment by jessegreenberg** [comment]
I made issues and assigned each to the responsible dev. This can be closed.

**Comment by zepumph** [comment]
Over in https://github.com/phetsims/projectile-motion/issues/216 @jessegreenberg said: 

> A scenery-phet/TimeControlNode was created that . . . includes PhET-iO

While converting, I found that there isn't `phetioDocumentation` in TimeControlNode, so doing the conversion would cause a loss of phetioDocumentation. Let's add some! Reopening and assigning.

**Comment by zepumph** [comment]
How does that look @jessegreenberg?

**Comment by jessegreenberg** [comment]
Contents of the documentation looks good to me.
